pkgs/tools/misc/trillian/gomod2nix.toml

```diff
@@ -0,0 +1,167 @@
+schema = 3
+subPackages = ["cmd/trillian_log_server", "cmd/trillian_log_signer", "cmd/createtree", "cmd/deletetree", "cmd/updatetree"]
```

Comment:
Why does this need to be moved?

Reply:
It's not moved per se, you can still set the same attribute in the Nix expression.
This is just another way to set the same thing which is a lot easier to control from the code generator.

Reply:
For one it serves to reduce the build time closure size.
We don't actually need everything in the root level go.mod and by making the generator aware of what needs to be built we only need to pull in the actually required subset of dependencies.